### PR TITLE
Update en.json

### PR DESCRIPTION
**Pull request by theyokohamalife**
Updates for the About page.

Screen Shot 2021-06-28 at 12 32 01 AM (2)

### CODE REVIEW

**Review by renderf0x — APPROVED** [review]
Awesome, I'm stoked we're adding more info about code contrib into the about page.

File: locales/en.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
     "about": {
         "creatorA": "This site was created and maintained by me, LaShawn Toyoda! If you have any trouble using the site or come across incorrect or outdated information, please contact me through Twitter {0}",
         "creatorQ": "Who created this website?",
-        "supportA": "The best way to help is by submitting any cancellation waiting lists you come across to the database. If you're feeling extra generous, please buy me a kofi!",
-        "supportQ": "How can I help support the database?",
-        "trustA": "All of the links lead to official resources such as clinic and government websites. Please read them thoroughly before registering your personal information with them.",
+        "supportA": "If you'd like to be part of the Find a Doc community, <a href='https://github.com/ourjapanlife'>join us over on GitHub</a> to see what we could use your help with!",
```
Comment (ann-kilzer) [inline locales/en.json:5]: Does this work? I think we need to use interpolated components:

https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage

File: locales/en.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
+        "supportQ": "How Can I Contribute?",
+        "trustA": "All of the links lead to official resources such as clinic and government websites. Please read them thoroughly before registering your personal information with them.<br /><br /> Please understand that because this database consists of content that can be submitted by anyone, it is not possible for us to guarantee that all of the information entirely accurate or up to date. <br /><br /> Our moderators try their best to make sure fraudulent information doesn't get published, but ask that you research any service that you find on here before visiting the location or submitting your personal information to a third-party website.<br /><br /> This is an open-source project, and we are not liable for any damages that may be incurred from the use of Find a Doc.<br /><br /><a href='https://github.com/ourjapanlife/findadoc-frontend/blob/f9c8af1dbc5f70f9196442f1170a86581f776400/LICENSE'> Read more about it in our license.</a>",
```
Comment (ann-kilzer) [inline locales/en.json:7]: same here

**Review by ann-kilzer — COMMENTED** [review]
Glad to see improved copy but unsure if raw html in the strings works.

See link in the comments

File: locales/en.json (hunk 1 above)
Comment (renderf0x) [reply locales/en.json:5]: You're 100% right Ann. I used this previously for our disclaimer here:
https://github.com/ourjapanlife/findadoc-frontend/blob/main/components/Disclaimer.vue#L3

It works, but we do need to be vigilant about making 100% sure when we're injecting HTML.

Comment (theyokohamalife) [reply locales/en.json:5]: I checked that it does work. I got the idea from the Disclaimer code @renderf0x wrote.

**Review by ann-kilzer — APPROVED** [review]
Let's try it and we can always switch to an interpolated component later to make translation easier for non-devs

File: locales/en.json (hunk 1 above)
Comment (ann-kilzer) [reply locales/en.json:5]: Ahhh cool. Let's merge, and I may submit a small PR later to separate the code from the text

**Pull request merged**